src/test/java/uk/gov/hmcts/reform/sscstaskconfiguration/dmn/CamundaTaskPermissionTest.java

```diff
@@ -228,7 +228,49 @@ static Stream<Arguments> scenarioProvider() {
             Arguments.of(
                 "reviewRoboticFail",
                 "someCaseData",
-                Permissions.defaultCtscPermissions()
+                List.of(
+                    Permissions.DEFAULT_CASE_ALLOCATOR_PERMISSIONS,
+                    Permissions.DEFAULT_TASK_SUPERVISOR_PERMISSIONS,
+                    Permissions.ALLOCATED_TRIBUNAL_CASEWORKER_READ_PERMISSIONS,
+                    Permissions.TRIBUNAL_CASEWORKER_READ_PERMISSIONS,
+                    Map.of(
```

Comment:
Can use the following to create the map:
`permission("allocated-ctsc-caseworker","Read,Own,Claim,Unclaim,Manage,UnclaimAssign,CompleteOwn", "CTSC", 1)`